src/Amazon.CloudWatch.EMF/Model/StorageResolution.cs

```diff
@@ -0,0 +1,8 @@
+namespace Amazon.CloudWatch.EMF.Model
+{
+    public enum StorageResolution
+    {
+        STANDARD = 60,
+        HIGH = 1
```

Comment:
If I'm not mistaken enum naming should follow standard type-naming rules and therefore be in PascalCasing.

Reply:
Unit Enum is following the same style so just to be in consistent followed the same .